apps/backend/src/libs/plugins/authorization-plugin/authorization-plugin.ts

```diff
@@ -8,6 +8,7 @@ import { AuthError } from "~/modules/auth/auth.js";
 import { type UserService } from "~/modules/users/users.js";
 
 import { ServerHooks } from "../libs/enums/enums.js";
+import { ONE_THOUSAND_MILISECONDS } from "./libs/constants/constants.js";
```

Comment:
```suggestion
import { ONE_THOUSAND_MILLISECONDS } from "./libs/constants/constants.js";
```

Reply:
What spellchecker do you use? My doesn't give any suggestions

Reply:
I don't use any spellchecker, I just see the mistake :)